### DISCUSSION TITLE
How to completely stop revalidating?

### DISCUSSION THREAD

**Discussion by lukasrakauskas** [vercel/swr#624] [General] [answered]
Hello,
I have a single resource that I'd like to update while it exists. Unfortunately, at some point, it gets deleted and SWR still tries to update and makes unnecessary 404 requests. I am aware that swr uses exponential backoff by default and that works in favour, but I want to stop those requests completely. So my question is there a way to delete a key from revalidating forever? Do I have to write a custom onErrorRetry function or is there a different/better way?

**Comment by sergiodxa** [comment, marked as answer]
Usually if you don't have a need for a certain key you unmount the component using SWR with that key, and if there is no mounted component using a key SWR will not revalidate it.

**Reply by lukasrakauskas** [reply]
Thanks, I managed to get it working by extracting the hook from page component to a smaller component.
